src/lib/layers/ContextualLayers.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
```

Comment:
Making room for more map-wide overlays here! The hard part, IMO, is UX decisions raised in the PR description about disabling other layers/controls while one of these is active.